moped-editor/src/views/dashboard/DashboardTimelineModal.js

```diff
@@ -53,17 +52,20 @@ const DashboardPhaseModal = ({
         maxWidth={"xl"}
       >
         <DialogTitle disableTypography className={classes.dialogTitle}>
-          <h4>{`Update phase - ${project.project_name}`}</h4>
+          <h4>{`Update phase - ${projectName}`}</h4>
```

Comment:
This needs to toggle based on the table name

Reply:
Thank you for catching this!! Fixed